virttest/qemu_devices/qdevices.py

```diff
@@ -729,17 +759,7 @@ def clear_child_nodes(self):
         self._child_nodes.clear()
 
     @staticmethod
-    def _convert_blkdev_args(args):
-        """
-        Convert string type of 'on' and 'off' to boolean, and create new dict
-        (e.g: 'cache': {'direct': 'true'}) from key which include symbol '.'
-        (e.g: 'cache.direct': 'true') to adhere to the blockdev qmp syntax.
-
-        :param args: Dictionary with the qmp parameters.
-        :type args: dict
-        :return: Converted args.
-        :rtype: dict
-        """
+    def _convert_raw_style_to_json_style_args(args):
```

Comment:
Suggest keeping the previous name until we have a better approach for that, because the actual procedure being called here is about parsing the given params to get desired arguments rather than doing the conversion from raw to json.